src/test/groovy/no/skatteetaten/aurora/boober/service/GitServiceTest.groovy

```diff
@@ -26,6 +26,8 @@ class GitServiceTest extends Specification {
     userDetailsProvider.getAuthenticatedUser() >> new User("aurora", "token", "Aurora Test User", [])
   }
 
+  @Ignore
+  //does not work on mac
```

Comment:
I would add the description of why the test is ignore in @Ignore annotation. Makes the visibility better when running the tests in for example idea of the reason behind it.

Reply:
Thanks!